### PR TITLE
This is HW_4

### PR DESCRIPTION
**Pull request by azartnyi**
Hello, check please <3
image

### CODE REVIEW

File: HW_4/html+js/hw4_3.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+let minAge = 18;
```
Comment (VladimirLQA) [inline HW_4/html+js/hw4_3.js:1]: Use `const` whenever possible

File: HW_4/hw4_1_2.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//Task 1
```
Comment (VladimirLQA) [inline HW_4/hw4_1_2.js:1]: Add descriptions for tasks

File: HW_4/hw4_1_2.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    console.log("WellCum!");
+}
+else if (age > maxAge) {
+    console.log("Keep calm and look Culture channel");
+}
+else {
+    console.log("Technical work, guys");
+}
+
+//Task 2
```
Comment (VladimirLQA) [inline HW_4/hw4_1_2.js:19]: Some tasks can have subtasks with modifications to previous !

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: HW_4/hw4_1_2.js (hunk 2 above)
Comment (azartnyi) [reply HW_4/hw4_1_2.js:1]: fixed

File: HW_4/hw4_1_2.js (hunk 3 above)
Comment (azartnyi) [reply HW_4/hw4_1_2.js:28]: fixed

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**